frontend-zs/src/components/stories/StoryPage.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Media, Container } from 'react-bootstrap'
+import Axios from 'axios'
+import { Link } from 'react-router-dom'
+import { TiArrowBack } from "react-icons/ti"
+import StoryDetails from './StoryDetails'
+import StoryPagination from './StoryPagination'
+import StoryQuestionLike from './StoryQuestionLike'
+import { authenticationService } from '../../services/authentication.service';
+import './Story.scss'
+
+function goBack() {
+    return (
+        <Button className="btn-back mb-3" size="sm" active>
+            <Link to="/stories" >
+                <Media>
+                    <TiArrowBack size={20} />
+                    <Media.Body>Back</Media.Body>
+                </Media>
+            </Link>
+        </Button>
+    )
+}
+
+function StoryText(displayStory) {
+    return (
+        <div className="p-4 new-line" style={{ fontSize: 19 }} >
+            {displayStory}
+        </div>
+    )
+}
+
+export default function StoryPage(props) {
+    const [story, setStory] = useState({
+        title: '',
+        author: '',
+        authorImage: '',
+        storyPages: [],
+        categories: [],
+        numberLikes: []
+    })
```

Comment:
```suggestion
    const [story, setStory] = useState({})
```

Reply:
I defined the default state like that because otherwise I get the error below. This happens because the page is monted before the data is fetched. 

![image](https://user-images.githubusercontent.com/29261118/80300371-d4097f80-87a4-11ea-84ad-1f430e2f9c15.png)

Reply:
This could be handled by changing it to `{(story.numberLikes?.length) ?? 0}` but you'd have to handle this it for a lot of parts of the code as well, so if you don't feel like changing it just resolve this.

Reply:
I fixed it.